compressed_image_transport/src/compressed_publisher.cpp

```diff
@@ -51,29 +53,37 @@ namespace enc = sensor_msgs::image_encodings;
 namespace compressed_image_transport
 {
 
-void CompressedPublisher::advertiseImpl(ros::NodeHandle &nh, const std::string &base_topic, uint32_t queue_size,
-                                        const image_transport::SubscriberStatusCallback &user_connect_cb,
-                                        const image_transport::SubscriberStatusCallback &user_disconnect_cb,
-                                        const ros::VoidPtr &tracked_object, bool latch)
```

Comment:
tracked_object was removed from the arguments, just checking that it was on propose.

Reply:
Correct, it was removed in the image_transport as well.